Controller/Controller.php

```diff
@@ -164,10 +165,16 @@ public function getTranslationsAction(Request $request, $domain, $_format)
             }
         }
 
+        if (version_compare(Kernel::VERSION, '2.7.0', '>=')) {
```

Comment:
@stof Thanks for this point, didn't thought about that. So now it is method_exists().